client/http-client.js

```diff
@@ -10,8 +10,13 @@ module.exports = (clientConfig) => {
     const HttpsProxyAgent = require('https-proxy-agent')
     agent = new HttpsProxyAgent(clientConfig.proxy)
   } else if (![null, false].includes(clientConfig.agent)) {
-    const http = require('http')
-    agent = new http.Agent({keepAlive: true})
+    if (/^https/i.test(clientConfig.url)) {
```

Comment:
the case sensitive config is a bit weird here 😁

Reply:
true that! 😂

Reply:
FYI node-fetch doesn't support the http agent properly. They only have one, so redirects won't work across domains. Just something we have to keep in mind in case the issue comes up (which shouldn't).